.github/workflows/k8s-patch-os-jobs.yml

```diff
@@ -0,0 +1,173 @@
+# The EC2 AMI for K8s needs to be updated to prevent it from becoming outdated. This workflow will run in the beginning of every month to replace the existing
+# instances with new ones
+name: Update K8s EC2 Instance
```

Comment:
This and the other job have the same name? Be more descriptive but stay concise.